clients/pyethereum:develop/Dockerfile

```diff
@@ -9,6 +9,11 @@ RUN \
   python get-pip.py &&                                                         \
   git clone --depth 1 https://github.com/ethereum/pyethapp  && \
   cd pyethapp && python setup.py develop                                                   && \
```

Comment:
If you don't put this line in paranthesis, the directory change will be in effect for all subsequent commands, so the one below won't actually change.